x-pack/plugins/security_solution/public/entity_analytics/components/asset_criticality_file_uploader/components/schedule_risk_engine_callout.tsx

```diff
@@ -0,0 +1,126 @@
+  const { isLoading: isLoadingRiskEngineSchedule, mutate: scheduleRiskEngineMutation } =
+    useScheduleNowRiskEngineMutation({
+      onSuccess: () =>
+        addSuccess(
+          i18n.translate(
+            'xpack.securitySolution.entityAnalytics.assetCriticalityResultStep.riskEngine.successMessage',
+            {
+              defaultMessage: 'Risk engine run scheduled',
+            }
+          )
+        ),
+      onError: (error) =>
+        addError(error, {
+          title: i18n.translate(
+            'xpack.securitySolution.entityAnalytics.assetCriticalityResultStep.riskEngine.errorMessage',
+            {
+              defaultMessage: 'Risk engine schedule failed',
+            }
+          ),
+        }),
+    });
+
+  const { status, runAt } = riskEngineStatus?.risk_engine_task_status || {};
+
+  const isRunning = useMemo(
+    () => status === 'running' || (!!runAt && new Date(runAt) < new Date()),
+    [runAt, status]
+  );
+
+  const countDownText = useMemo(
+    () =>
+      isRunning
+        ? i18n.translate(
+            'xpack.securitySolution.entityAnalytics.assetCriticalityResultStep.riskEngine.nowRunningMessage',
+            {
+              defaultMessage: 'Now running',
+            }
+          )
+        : formatTimeFromNow(riskEngineStatus?.risk_engine_task_status?.runAt),
+    [isRunning, riskEngineStatus?.risk_engine_task_status?.runAt]
+  );
+
+  const scheduleRiskEngine = useCallback(() => {
+    scheduleRiskEngineMutation();
+  }, [scheduleRiskEngineMutation]);
+
+  if (!riskEngineStatus?.isNewRiskScoreModuleInstalled) {
+    return null;
+  }
+
+  return (
+    <EuiCallOut
+      data-test-subj="risk-engine-callout"
+      title={
+        <FormattedMessage
+          defaultMessage="Risk score"
+          id="xpack.securitySolution.entityAnalytics.assetCriticalityResultStep.riskEngine.calloutTitle"
+        />
+      }
+      color="primary"
+      iconType="iInCircle"
+    >
+      <FormattedMessage
+        defaultMessage="The assigned criticalities will influence the calculated risk score on the next engine run."
+        id="xpack.securitySolution.entityAnalytics.assetCriticalityResultStep.riskEngine.calloutText"
+      />
+      <EuiHorizontalRule />
+      <EuiFlexGroup direction="row">
+        <EuiFlexItem>
+          <EuiText size="xs">
+            <FormattedMessage
+              defaultMessage="The next scheduled engine run is in:"
+              id="xpack.securitySolution.entityAnalytics.assetCriticalityResultStep.riskEngine.scheduleText"
+            />
+            <b>{` ${countDownText}`}</b>
+          </EuiText>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiButtonEmpty
+            iconType="play"
+            size="xs"
+            onClick={scheduleRiskEngine}
+            isLoading={isLoadingRiskEngineSchedule || isRiskEngineStatusLoading || isRunning}
+          >
+            <FormattedMessage
+              defaultMessage="Run engine now"
```

Comment:
```suggestion
              defaultMessage="Recalculate entity risk scores now"
```
Optional suggestion – thought it may help to focus on the result rather than the action